### DISCUSSION TITLE
what does <leader>ue doing with edgy.nvim

### DISCUSSION THREAD

**Discussion by cStralpt** [LazyVim/LazyVim#926] [General]
i want to remap edgy window with C-b i try this:
```
keys = {
  -- hide window
  ["<c-b>"] = function(win)
    win:hide()
  end,
}
```
but it didn't do like what i expected.
expectation: do something similar like <leader>ue.
reason: smooth transition between vscode and neovim,

**Comment by folke** [comment]
That just hides a single window. Check the lazyvim spec for edgy to see how you can do what leader-ue does
